windows/ReactNativeWebView/ReactWebView2.cpp

```diff
@@ -106,6 +136,35 @@ namespace winrt::ReactNativeWebView::implementation {
         }
     }
 
+    void ReactWebView2::HandleMessageFromJS(winrt::hstring const& message) {
+        winrt::JsonObject jsonObject;
+        if (winrt::JsonObject::TryParse(message, jsonObject) && jsonObject.HasKey(L"type")) {
+            auto type = jsonObject.GetNamedString(L"type");
+            if (type == L"__alert") {
+                auto dialog = winrt::MessageDialog(jsonObject.GetNamedString(L"message"));
+                dialog.Commands().Append(winrt::UICommand(L"OK"));
```

Comment:
I think this probably should be localized so it shows up in the right language. IIRC XAML already has these strings so you can probably use those

Reply:
This is the structure of the original WebView code. Seems like a good change to make though. Could you clarify what you mean by "XAML already has these strings so you can probably use those"

Reply:
Will resolve later in a fresh PR.